flink-doris-connector/src/main/java/org/apache/doris/flink/cfg/DorisConnectionOptions.java

```diff
@@ -27,19 +27,30 @@ public class DorisConnectionOptions implements Serializable {
 
     private static final long serialVersionUID = 1L;
 
-    protected final String fenodes;
+    protected String fenodes;
     protected final String username;
     protected final String password;
     protected String jdbcUrl;
+    protected String beNodes;
+    protected boolean enableIntranetAccess;
```

Comment:
Is it possible not to add this parameter? When passing benodes, can streamload directly use benodes?

Reply:
This is a good suggestion